velox/functions/prestosql/tests/JsonExtractScalarTest.cpp

```diff
@@ -1,165 +0,0 @@
-/*
- * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Why did you delete this file and combine the tests ? Isn't it better to retain this file ?